Hw19/ManchkinGame/ModifierDeck.cpp

```diff
@@ -0,0 +1,56 @@
+
+#include "ModifierDeck.h"
+
+#include <algorithm>
+
+#include "Modifier.h"
+#include "MyAlgorithm.h"
+
+ModifierDeck::ModifierDeck()
+{
+	m_modifiersDatabase.emplace_back(new SimpleModifier{ 3 });
```

Comment:
Remove it then since you don't use it anymore

Reply:
No!
It is used as start cards.